database/migrations/2018_07_03_091314_create_ports_stack_table.php

```diff
@@ -13,11 +13,14 @@
     public function up(): void
```

Comment:
please do not change historical migrations

Reply:
I wish I could avoid changing historical migrations.